libavfilter/dnn/dnn_io_proc.c

```diff
@@ -27,6 +27,12 @@
 int ff_proc_from_dnn_to_frame(AVFrame *frame, DNNData *output, void *log_ctx)
 {
     struct SwsContext *sws_ctx;
+    int frame_size = frame->height * frame->width;
+    int linesize[3];
+    linesize[0] = frame->linesize[0];
```

Comment:
why just set the linesize[0]? yuv420p should have 3 planes.

Reply:
Actually, the SR model supported in openvino & tensorflow backend, only accept Y  channel, the U/V channel was scaled up by FFmpeg software scale.
Only libtorch SR model, which is the model we are working on, scale all three channels of RGB of image.
But you do alert me, I will check if the OpenVINO&Tensorflow will be influenced by this change.